styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -0,0 +1,35 @@
+{% include "@atoms/skip-to-main-content.twig" %}
+
+{% block ribbon %}
+  {% include "@organisms/ribbon.twig" %}
+{% endblock %}
+
+{% block header %}
+  {% include "@organisms/header.twig" %}
+{% endblock %}
+
+<main id="main-content" class="three-column" tabindex="-1" role="main">
```

Comment:
This is missing the main container class that keeps the layout from stretching too wide on larger screens.